File: services/ms-cliente/src/main/java/airportsystem/mscliente/repository/ClienteRepository.java
package airportsystem.mscliente.repository;

import airportsystem.mscliente.model.Cliente;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ClienteRepository extends JpaRepository<Cliente, String> {

    Cliente findByEmail(String email);
    Cliente findByCpf(String cpf);

    boolean existsByEmail(String email);
}

File: services/ms-cliente/airport-system.ms-cliente/src/main/kotlin/com/tads/airport_system/ms_cliente/repository/ClienteRepository.kt
package com.tads.airport_system.ms_cliente.repository

import com.tads.airport_system.ms_cliente.model.Cliente
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface ClienteRepository : JpaRepository<Cliente, String> {
    fun findByEmail(email: String): Cliente?
    fun existsByEmail(email: String): Boolean
} 